flink-libraries/flink-cep-scala/src/main/scala/org/apache/flink/cep/scala/pattern/Pattern.scala

```diff
@@ -271,7 +271,7 @@ class Pattern[T , F <: T](jPattern: JPattern[T, F]) {
     * {{{A1 A2 B}}} appears, this will generate patterns:
     * {{{A1 B}}} and {{{A1 A2 B}}}. See also {{{allowCombinations()}}}.
     *
-    * @return The same pattern with a [[Quantifier.ONE_OR_MORE()]] quantifier applied.
+    * @return The same pattern with a [[Quantifier.oneOrMore()]] quantifier applied.
```

Comment:
I hesitated with this change as it is, as i understand it, an API breaking change. @kl0u What's your take on this?